lib/java/com/google/android/material/tabs/TabLayoutMediator.java

```diff
@@ -238,14 +241,16 @@ void reset() {
    */
   private static class ViewPagerOnTabSelectedListener implements TabLayout.OnTabSelectedListener {
     private final ViewPager2 viewPager;
+    private final Boolean smoothScroll;
```

Comment:
Any reason for this to be a Boolean rather than a boolean?

Reply:
this is clearly a mistake, coming from Kotlin's background. I will correct this .